diff --git a/lib/sharp.js b/lib/sharp.js
index 1081c93..3e6e5e0 100644
--- a/lib/sharp.js
+++ b/lib/sharp.js
@@ -22,6 +22,15 @@
 let path, sharp;
 const errors = [];
+if (runtimePlatform === 'win32-x64') {
+  path = '@img/sharp-win32-x64/sharp.node';
+  try {
+    sharp = require('@img/sharp-win32-x64/sharp.node');
+  } catch (err) {
+    errors.push(err);
+  }
+}
 for (path of paths) {
+  if (sharp) break;
   try {
     sharp = require(path);
     break;
